From 27acab928d42dc2e99bd58a8a867bf8758a28848 Mon Sep 17 00:00:00 2001
From: orthos <orthos@huawei14.arch.suse.de>
Date: Thu, 2 Dec 2021 14:35:39 +0100
Subject: [PATCH] use cmark instead of pandoc

cmark has much less build dependencies and already available for SLE

Signed-off-by: Mian Yousaf Kaukab <ykaukab@suse.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ee669ae..86f89d7 100644
--- a/Makefile
+++ b/Makefile
@@ -51,7 +51,7 @@ restool: $(OBJ)
 	$(CC) $(CFLAGS) -c $^ -o $@
 
 %.1: %.md
-	pandoc --standalone --to man $^ -o $@
+	cmark -t man $^ > $@
 
 install: restool scripts/ls-main scripts/ls-append-dpl scripts/ls-debug scripts/restool_completion.sh $(MANPAGE)
 	install -D -m 755 restool $(DESTDIR)$(bindir)/restool
-- 
2.31.1

